packages/keyring-api/src/events.ts

```diff
@@ -11,3 +17,73 @@ export enum KeyringEvent {
   RequestApproved = 'notify:requestApproved',
   RequestRejected = 'notify:requestRejected',
 }
+
+export const AccountCreatedEventStruct = object({
```

Comment:
Those are now part of the public API (they were in the `internal/` folder).

Reply:
Not against this change. The reasoning for not having it in the public API was that only the `SnapKeyring` was consuming it.